crt_portal/cts_forms/templates/forms/complaint_view/index/filters/violation_summary_filter.html

```diff
@@ -9,7 +9,4 @@
     <span class="usa-sr-only">personal description</span>
```

Comment:
This file change is probably not required. This element is not part of 622. This may get developed in near future. May be we should revert this, or leave a comment.